admin_manual/appliance/firewall.rst

```diff
@@ -0,0 +1,19 @@
+Firewall protected environment
+------------------------------
+
+If you are considering setting up the appliance in an environment with a firewall, 
+please create rules that permit access to the following hosts. 
+If your DNS is not working, you can use the IP addresses instead.
+If you are using google as your DNS server(IP=8.8.8.8), you have to permit access to it too.
```

Comment:
s/google/Google/g

Reply:
DNS server(IP=8.8.8.8) --> DNS server (IP=8.8.8.8)
new: the blank between server and (